Jenkinsfile

```diff
@@ -248,6 +248,8 @@ stage('Unit Test') {
         unpack_lib('gpu', tvm_multilib)
         timeout(time: max_time, unit: 'MINUTES') {
           sh "${docker_run} ${ci_gpu} ./tests/scripts/task_ci_python_setup.sh"
+          sh "${docker_run} ${ci_gpu} ./tests/scripts/task_config_build_gpu.sh"
+          sh "${docker_run} ${ci_gpu} ./tests/scripts/task_build.sh build standalone_crt"
```

Comment:
Would this trigger rebuild of libtvm? or simply just `standalone_crt`?

Reply:
ok resolved per our discussion. we will not explicitly introduce a separate build step here; instead we will add this build to `tests/scripts/task_ci_setup.sh`, which was previously `tests/scripts/task_ci_python_setup.sh`